docs/tableOfContents.js

```diff
@@ -258,19 +258,19 @@ export default [
     autoSort: true,
     pages: [
       new Page({
-        path: '/KThemePlugin',
+        path: '/kthemeplugin',
```

Comment:
I hope you automated this somehow instead of typing every one of these out :fearful:

Reply:
yup - a little regex and a little sublime text magic :D